soundfile.py

```diff
@@ -968,6 +990,120 @@ def read(self, frames=-1, dtype='float64', always_2d=False,
                 out[frames:] = fill_value
         return out
 
+    def read_raw(self, frames=-1, dtype=None):
+        """\ 
+        Read raw audio data from the audio file (not to be confused with reading
+        RAW header-less PCM files).
+
+        Note : The result of using of both regular reads/writes and raw
+        reads/writes on compressed file formats other than ALAW and ULAW is
+        undefined.
+
+        Parameters
+        ----------
+        frames : int, optional
+            The number of frames to read. If `frames < 0`, the whole
+            rest of the file is read.
+        dtype : {'float64', 'float32', 'int32', 'int24', 'int16', 'int8', 'uint8'}
+            Audio data sample format.
+
+        Returns
+        -------
+        buffer
+            A buffer containing the read data.
```

Comment:
I do not want pysoundfile to leak CFFI data structures. I would much prefer a `bytes` or `bytearray` object instead of a cffi buffer.